app/src/main/kotlin/com/twain/interprep/presentation/ui/components/generic/IPLargeAppBar.kt

```diff
@@ -370,17 +370,6 @@ fun TodayInterviewCard(
                             interview.time
                         ))
                     }
-                    if (interview.meetingLink.isNotEmpty()) {
```

Comment:
If we remove this, how are we making "Join here" as a hyperlink with meeting link?